client/src/redux/sound-mode-saga.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable require-yield */
+
+import { takeEvery } from 'redux-saga/effects';
+import store from 'store';
+
+const soundKey = 'fcc-sound';
+
+export function setSound(setting) {
+  store.set(soundKey, setting);
+}
+
+function* updateLocalSoundSaga({ payload: { sound } }) {
+  return setSound(sound);
+}
```

Comment:
```suggestion
function* updateLocalSoundSaga({ payload: { user, sound } }) {
  if (user) {
    const { sound = false } = user;
    return setSound(sound);
  }
  return setSound(sound);
```
Without this the user's settings get ignored and it has to rely on the store.